### PR TITLE
WIP for Issue #4, Example API Call for Creating an Action

### PR DESCRIPTION
**Pull request by paulcastro**
This PR provides an example of how to make an API call using the Go whisk package, which provides a full implementation of the OpenWhisk REST API.

In examples/exampleApi.go, we make an API call to create a Swift 3 HelloWorld action.

The flow is:
1. Create a whisk client which is a container for an http client and specific entity operations.  You access these operations using `client.<entityname>.<operation> `.  For example, to create an action we do client.Actions.Insert( .... )
2. Fill out an Action struct that represents the Whisk Action.
3. Call client.Actions.Insert(action, false, false)  where `action` is the Action struct and the booleans represent sharedSet and overwrite, respectively.

Other commands follow this basic model.

### CODE REVIEW

File: examples/exampleApi.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package examples
+
+import (
+  "fmt"
+  "net/url"
+  "net/http"
+  "github.com/openwhisk/go-whisk/whisk"
```
Comment (lzbj) [inline examples/exampleApi.go]: is this package exists, go get complaints not find the repo.

File: examples/exampleApi.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+)
+
+func testCreateAction() {
```
Comment (lzbj) [inline examples/exampleApi.go]: I suggest we put the test code into separate file, we better obey go best practices

File: examples/exampleApi.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  createAction("helloWskTool", "func main(args: [String:Any]) -> [String:Any] { return [\"msg\":\"wsktool says hello\"] }", "swift:3")
+}
+
+func createAction(name string, code string, kind string) {
```
Comment (lzbj) [inline examples/exampleApi.go]: is it better we put this func to the utiliy package?

File: examples/exampleApi.go [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+  // call ActionService Thru Client
+  _, _, err = client.Actions.Insert(action, false, false)
+  if err != nil {
+    fmt.Println("Got error inserting action ", err)
+  }
+}
+
+// Utility to convert hostname to URL object
+func getURLBase(host string) (*url.URL, error)  {
```
Comment (lzbj) [inline examples/exampleApi.go]: same as above.

File: examples/exampleApi.go (hunk 1 above)
Comment (paulcastro) [inline examples/exampleApi.go]: Yes, sorry I copied this manually.

File: examples/exampleApi.go (hunk 2 above)
Comment (paulcastro) [inline examples/exampleApi.go]: This is badly named (by me).  This isn't really a test, just an example of how to use the API.

File: examples/exampleApi.go (hunk 3 above)
Comment (paulcastro) [inline examples/exampleApi.go]: Yes we will need to refactor.

**Pull request merged**